xtest/conftest.py

```diff
@@ -271,3 +278,311 @@ def attribute_two_kas_grant_and(
     otdfctl.grant_assign_value(kas_entry_beta, bet)
 
     return allof
+
+
+@pytest.fixture(scope="module")
+def one_attribute_attr_kas_grant(
+    otdfctl: abac.OpentdfCommandLineTool,
+    kas_url2: str,
```

Comment:
We may want to create more KASes to allow us to inspect which level a grant was assigned for, and then not decrypt the ones that won't be able to be decrypted (or try and assert that they failed to decrypt as expected).

For example, we could have a kas called like http://namespace.kas which is granted to the namespace, and assert when we expect that to be assigned.

Reply:
true ya to verify the specificity unit tests, ill add this in a follow up PR